Doc/library/http.server.rst

```diff
@@ -24,7 +24,7 @@ This module defines classes for implementing HTTP servers.
 
 One class, :class:`HTTPServer`, is a :class:`socketserver.TCPServer` subclass.
 It creates and listens at the HTTP socket, dispatching the requests to a
-handler.  Code to create and run the server looks like this::
+handler. Code to create and run the server looks like this::
```

Comment:
Please avoid this kind of changes.  They do not fix or improve anything, so just churn history for not benefit.